addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitSettings.java

```diff
@@ -140,6 +150,26 @@ public String getThermostatOffMode() {
         return thermostatOffMode;
     }
 
+    public String getGarageDoorOpenState() {
+        return garageDoorOpenState;
+    }
+
+    public String getGarageDoorClosedState() {
+        return garageDoorClosedState;
+    }
+    
+    public String getGarageDoorOpeningState() {
+        return garageDoorOpeningState;
+    }
+
+    public String getGarageDoorClosingState() {
+        return garageDoorClosingState;
+    }
+    
+
```

Comment:
Please format using eclipse esh profile some new lines are missing and some are duplicated.